### ISSUE TITLE
Penny and shield are out of sync.

### ISSUE THREAD

**Issue by BrucePerens** [GrottoPress/penny#4]
I'm running from Shield master until you make your next release. Penny is out of sync with master. It would be nice if you would keep them in sync. This will probably be less of a concern after your next release.

**Comment by akadusei** [comment]
Will be releasing soon. For now, if you are worried about anything breaking, may I suggest you pin *Shield* to a specific commit, rather than to the `master` branch:

```yaml
# ->>> shard.yml

dependencies:
# ...
  shield:
    github: GrottoPress/shield
    commit: f2d9f7ad210f6db2a198443102f149e9d4e8a6e6
# ...
```

**Comment by akadusei** [comment]
Resolved by be23e9a134081e794ac675f717ce831f2927dd02
